packages/lit-virtualizer/src/layouts/shared/BaseLayout.ts

```diff
@@ -409,6 +409,7 @@ export abstract class BaseLayout<C extends BaseLayoutConfig> implements Layout {
   protected _emitScrollSize() {
     const detail = {
       [this._sizeDim]: this._scrollSize,
+      [this._secondarySizeDim]: null
```

Comment:
Not following this change exactly?

Reply:
Probably best understood by looking at the whole commit in isolation: https://github.com/PolymerLabs/uni-virtualizer/pull/134/commits/9dd0bf249cf574c23cda6ce2420faf30f5edd6c1

In a nutshell, virtualizer is intended to be agnostic to whether a given layout is fundamentally working in just one dimension (either horizontally or vertically, like all of our current layouts) or in two dimensions (imagine a future layout that virtualizes in both dimensions, like Google Maps).

However, before this change, the implementation didn't reflect this intention; virtualizer was assuming the layout would always provide either a height or a width, but not both. It now requires both, but allows the layout to say it doesn't care about one or the other by setting that value to `null`.

Reply:
It makes sense! Thank you!